packages/thumbprint-react/components/Switch/index.module.scss

```diff
@@ -0,0 +1,53 @@
+@import '~@thumbtack/thumbprint-tokens/dist/scss/_index';
+
+// Match the width/height of Thubmprint Checkbox
+$bar-height: 20px;
+$bar-height-large: 28px;
+$height-width-ratio: 1.8;
+$transition-duration: 0.1s;
```

Comment:
Should we leverage the `$tp-duration__1` or `$tp-duration__2` token here instead? How did we come about selecting `0.1s` for the duration?